exonum-java-binding-core/src/test/java/com/exonum/binding/service/adapters/UserServiceAdapterTest.java

```diff
@@ -266,4 +270,19 @@ void afterCommit_ClosesCleaner() {
 
     assertTrue(cleaner.isClosed());
   }
+
+  @Test
+  void afterCommit_unexpectedException() {
```

Comment:
Would you please add a separate test for swallowing the exception — because the present (1) implicitly checks that — I'd recommend to `verify`; (2) is clutterred with logging assertions?